engine/src/main/java/org/terasology/rendering/assets/skeletalmesh/SkeletalMeshData.java

```diff
@@ -20,11 +20,11 @@
 import com.google.common.collect.Maps;
 import gnu.trove.list.TIntList;
 import gnu.trove.list.array.TIntArrayList;
+import org.joml.Quaternionf;
+import org.joml.Vector2f;
+import org.joml.Vector3f;
```

Comment:
Some uses of Vector3f are not equivalent in this file.

Reply:
what cases does this not work?

Reply:
I only see the change from `scale` to `mul`with scalar value, which looks correct to me.

Reply:
Clarification after discussion on Discord:

> e.g. Vector2f.sub(Vector2f, Vector2f) is present in both APIs but does something different in each
> one is dest.sub(minu, subtra) and the other is minu.sub(subtra, dest)

Would be good to compile a list of these incompatibilities and make sure to check them in the ported files.

Reply:
semantics of ll. 163-164 change